core/domain/exp_services.py

```diff
@@ -1101,6 +1399,16 @@ def get_next_page_of_all_commits(
     fetch_page() at:
 
         https://developers.google.com/appengine/docs/python/ndb/queryclass
+
+    Args:
+        page_size: int. Number of commits that are in the commit list page.
```

Comment:
At this level of the stack, we don't know what a "commit list page" is. I think the docstring here should be more like "the maximum number of commits to return."